aio_pika/robust_connection.py

```diff
@@ -133,10 +133,8 @@ def add_reconnect_callback(self, callback: Callable[[], None]):
                     await self.__cleanup_connection(e)
 
                     log.warning(
-                        "Connection attempt to \"%s\" failed "
-                        "and will be retried after %d seconds",
```

Comment:
Why did you remove this line? `_on_connection_close` doesn't log it while reconnecting..

Reply:
`self._cleanup_connection` will log same log.

Reply:
You mean here? https://github.com/mosquito/aio-pika/blob/1065083434c4ef3b45c653e30bc78e8637b8dad1/aio_pika/robust_connection.py#L77-L80
But it won't log while reconnecting due to https://github.com/mosquito/aio-pika/blob/1065083434c4ef3b45c653e30bc78e8637b8dad1/aio_pika/robust_connection.py#L63-L65

Reply:
Do you want to swap it?

Reply:
`_on_connection_close` will be called when every connection closed. In [this case](https://github.com/mosquito/aio-pika/blob/1065083434c4ef3b45c653e30bc78e8637b8dad1/aio_pika/robust_connection.py#L63-L65) connection is establishing and this condition needed only for `self.__cleanup_connection`.

Reply:
I thought you want to log about reconnection interval here https://github.com/mosquito/aio-pika/blob/1065083434c4ef3b45c653e30bc78e8637b8dad1/aio_pika/robust_connection.py#L77-L80
OK, but now after removal of lines about it I don't see how `self._cleanup_connection` would log it, - there is no any logging in it.

Reply:
What do you propose? A simple way is copying log from [here](https://github.com/mosquito/aio-pika/blob/1065083434c4ef3b45c653e30bc78e8637b8dad1/aio_pika/robust_connection.py#L77-L80) to `connect` function I guess.

Reply:
I suggest to save _"and will be retried after %d seconds"_ part of log within `connect` method. I don't see how it will be logged twice.